src/core/thread/mesh_forwarder.cpp

```diff
@@ -729,11 +729,6 @@ ThreadError MeshForwarder::SendMesh(Message &aMessage, Mac::Frame &aFrame)
     aFrame.SetSrcAddr(mMacSource.mShortAddress);
 
     // write payload
-    if (aMessage.GetLength() > aFrame.GetMaxPayloadLength())
-    {
-        fprintf(stderr, "%d %d\n", aMessage.GetLength(), aFrame.GetMaxPayloadLength());
-    }
-
```

Comment:
This fprintf appears to have been placed for diagnostic reasons.  It fails to compile for the platform I am targeting due to a lack of fprintf and stderr.